lua/ttt2/libraries/database.lua

```diff
@@ -79,6 +79,17 @@ local callbackIdentifiers = database.callbackIdentifiers
 -- General Shared functions
 --
 
+---
+-- Creates a databaseElement combining all infos necessary to make changes to the serverside sql database
```

Comment:
Is this table structure documented somewhere? searching for `databaseElement` sadly does not return any results in our codebase besides this.

Reply:
If functions `database.SetDefaultValue`, `database.SetValue`, `database.GetValue`, `database.RemoveChangeCallback`, `OnChange`, `IsValueReceived` and `ValueReceived` don't count, then no, it's not explicitly documented.

Reply:
No its not documented. I was just advised by @saibotk to use it for the the UI.